.github/workflows/third-party-library-changes.yml

```diff
@@ -0,0 +1,78 @@
+name: Third Party Library Changes
+on:
+  pull_request:
+    branches:
+      - trunk
+      - "3.[89]"
```

Comment:
We can remove this line. Only WP >= 4.1 is currently supported. Some other workflows still have 3.x listed, but it's just a remnant of the past that's more of a hassle to remove than to just leave.